INSTALL/INSTALL.ubuntu2204.md

```diff
@@ -189,15 +197,15 @@ And configure the database connection:
 # 3. Migrating MONARC DB
 
 ```bash
-$ php ./vendor/robmorgan/phinx/bin/phinx migrate -c module/Monarc/FrontOffice/migrations/phinx.php
-$ php ./vendor/robmorgan/phinx/bin/phinx migrate -c module/Monarc/Core/migrations/phinx.php
+sudo php ./vendor/robmorgan/phinx/bin/phinx migrate -c module/Monarc/FrontOffice/migrations/phinx.php
+sudo php ./vendor/robmorgan/phinx/bin/phinx migrate -c module/Monarc/Core/migrations/phinx.php
```

Comment:
I don't think sudo is needed for any php command execution.
It depends on the user that is used for the installation and the user has a read access to the project directories.

Reply:
Right !